network-api/networkapi/settings.py

```diff
@@ -538,10 +540,9 @@
 # Buyers Guide Rate Limit Setting
 BUYERS_GUIDE_VOTE_RATE_LIMIT = env('BUYERS_GUIDE_VOTE_RATE_LIMIT')
 
-# Detect if we're testing
+# Detect if we're in official testing mode
```

Comment:
Nitpicking after PR is merged: can you define official here? Does it mean CI or testing on dev machines?

Reply:
Neither - the only testing Django knows of is the kind that happens when you run `manage.py test`, which this lets you detect.